libs/frontend/feature-build/src/components/toolbar/ToolbarText.vue

```diff
@@ -271,13 +241,16 @@ watch(
 }
 
 .font-size {
-  width: 72px;
+  width: 76px;
```

Comment:
The text in the font-size input seems a little crowded when the unit is rem instead of px, so I added a little more space to it.